index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+<body class="container">
+    <header class="header">
+        <p class="name">Fernando Herrera</p>
+        <div>
+            <div class="hamb hamb1"></div>
+            <div class="hamb hamb2"></div>
+            <div class="hamb hamb3"></div>            
+        </div>     
+    </header>
+    <main >
+        <div class="text-container">
+            <p class="welcome">Hey There. I'm Fernando</p>
+            <p class="profession">I'm Software Developer</p>
+            <p class="xp">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+        </div>
+        <div class="social-media"> 
+            <a href="#">
+                <img class="github" src="/img/github.png" alt="github">
```

Comment:
- [ ] Hi you've done well with all your other icon images 👏👏👏  please ensure that this one is also working currently it is not working due to the fact that you have a slash at the front of the `src` path. please remove this slash on this one like you have on the rest of the `img` tags.
something like this will ensure that your link works correctly.
```suggestion
                <img class="github" src="img/github.png" alt="github">
```